includes/class-wc-ajax.php

```diff
@@ -2240,21 +2227,38 @@ private static function variation_bulk_adjust_price( $variations, $field, $opera
 	}
 
 	/**
-	 * Bulk action - Set Meta.
+	 * Bulk set convenience function.
 	 * @access private
 	 * @param array $variations
 	 * @param string $field
 	 * @param string $value
 	 */
-	private static function variation_bulk_set_meta( $variations, $field, $value ) {
+	private static function variation_bulk_set( $variations, $field, $value ) {
+		foreach ( $variations as $variation_id ) {
+			$variation = wc_get_product( $variation_id );
+			$variation->{ "set$field" }( wc_clean( $value ) );
```

Comment:
It would make more sense to pass the actual prop name than the _ prefixed meta key. You can add that here.